src/power_grid_model_io/converters/vision_excel_converter.py

```diff
@@ -20,32 +141,57 @@ class VisionExcelConverter(TabularConverter):
     """
 
     def __init__(self, source_file: Optional[Union[Path, str]] = None, language: str = "en"):
-        mapping_file = Path(str(DEFAULT_MAPPING_FILE).format(language=language))
+        mapping_file = Path(str(DEFAULT_MAPPING_FILE).format(language=str(language)))
```

Comment:
Interesting, why do you use the `str()` function? Is it to make sure that language is a string? I think we shouldn't (have to) do that. It's up to the user to supply the right arguments, in the correct data type.